packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFDataConvertersTests.m

```diff
@@ -98,14 +98,12 @@ - (void)testFWFWKFrameInfoDataFromWKFrameInfo {
 }
 
 - (void)testFWFNSErrorDataFromNSError {
-  NSError *error = [NSError errorWithDomain:@"domain"
-                                       code:23
-                                   userInfo:@{NSLocalizedDescriptionKey : @"description"}];
+  NSError *error = [NSError errorWithDomain:@"domain" code:23 userInfo:@{@"a" : @"b"}];
```

Comment:
Related to my comment about crashing if the filtering were wrong in the code: it would be good to also include something here that won't work, to ensure that it's filtered out. E.g., `@"unsupported value": [NSData data]`.